src/main/java/Duke.java

```diff
@@ -1,10 +1,350 @@
+                    String[] parts2 = nameTime.split(" \\(by: ");
+                    String ddlName = parts2[0];
+                    String ddlTimeStr = parts2[1].substring(0, parts2[1].length() - 1);
+                    DateTimeFormatter formatterDdl = DateTimeFormatter.ofPattern(
+                            "EEE, dd/MMM/yyyy hh:mm a");
+                    LocalDateTime dateTime = LocalDateTime.parse(ddlTimeStr, formatterDdl);
+                    Deadline ddlToAdd = new Deadline(ddlName, dateTime);
+                    if (toAdd.charAt(4) == 'X') {
+                        ddlToAdd.setDone();
+                    }
+                    inputs.add(ddlToAdd);
+                }
+                // case Event
+                if (toAdd.charAt(1) == 'E') {
+                    String[] parts1 = toAdd.split("] ");
+                    String nameTime = parts1[1];
+                    String[] parts2 = nameTime.split(" \\(at: ");
+                    String eveName = parts2[0];
+                    String eveTimeStr = parts2[1].substring(0, parts2[1].length() - 1);
+                    DateTimeFormatter formatterEve = DateTimeFormatter.ofPattern(
+                            "EEE, dd/MMM/yyyy hh:mm a");
+                    LocalDateTime dateTime = LocalDateTime.parse(eveTimeStr, formatterEve);
+                    Event eveToAdd = new Event(eveName, dateTime);
+                    if (toAdd.charAt(4) == 'X') {
+                        eveToAdd.setDone();
+                    }
+                    inputs.add(eveToAdd);
+                }
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("data.txt is not found.");
+        }
+
+//        for (int i = 0; i < inputs.size(); i++) {
+//            System.out.println("test read todo and ddl\n" + dataRead.get(i));
+//        }
+
+        String dataFileStr = "./data/data.txt";
+
+        // Welcome message
+        System.out.println(lineBreak
+                + "\n"
+                + "Hello! I'm Azure.\n"
+                + "You can create Tasks here.\n"
+                + "To add a Todo, simply key in `todo` followed by the name;\n"
+                + "To add a Deadline, simply key in `deadline` followed by its name, `/by` and the deadline;\n"
+                + "To add an Event, simply key in `event` followed by its name, `/at` and its time.\n"
+                + "Please key in the date and time in the format of `26/08/2021 20:20`.\n"
+                + "How can I help you today?\n"
+                + lineBreak + "\n");
+
+        Scanner myObj = new Scanner(System.in);
+
+        // String input
+        while(true) {
+            String input = myObj.nextLine();
+            Task newTask = new Task(input);
+
+            // mark a task as Done
+            Pattern pattern = Pattern.compile("done\\s\\d+");
+            Matcher matcher = pattern.matcher(input);
+            boolean matchFound = matcher.find();
```

Comment:
```suggestion
            boolean ifMatchFound = matcher.find();
```
so that it sounds more like a boolean